packages/components/pie-card-container/src/card-container.scss

```diff
@@ -95,6 +95,13 @@
     &:focus-within {
         box-shadow: 0 0 0 2px var(--dt-color-focus-inner), 0 0 0 4px var(--dt-color-focus-outer);
     }
+
+    &.isDraggable {
+        a,
+        button {
```

Comment:
this should be &:not([interactiontype='none']) once this is merged

https://github.com/justeattakeaway/pie/pull/791/files